jeremy2918/tsseract
# Avatars

# [Avatars](https://trello.com/c/b2LuT67x/64-i-want-the-users-to-be-able-to-build-their-avatars-and-come-up-with-a-name-for-them)


This PR adds the user the possibility to create a custom avatar that they will use as a profile picture. The users can change their avatars whenever they want and will be able to customize them with a wide variety of different items to choose from.

## Testing Steps

- Pull and checkout to this branch.
- `npm install`
- `npm run tsc`
- `npm test` All tests should be passing successfully.
- `npm run watch`
- Go to the `/avatar` page and you should be able to see the avatar editor
- You can choose different items and colors for those items.
- You should be able to see the avatar as your profile picture and see the others' avatars as well.



## UI Changes
TO BE ATTACHED
